cmd/minikube/cmd/start_flags.go

```diff
@@ -110,6 +110,7 @@ const (
 	network                 = "network"
 	startNamespace          = "namespace"
 	trace                   = "trace"
+	userFlag                = "user"
```

Comment:
if this is a global flag, it doesnt need to be in start.go maybe in root.go ?

Reply:
Moved